tips/TIP-0048/openapi3-core.yaml

```
@@ -0,0 +1,3520 @@
+                $ref: '#/components/schemas/NotFoundResponse'
+        '500':
+          description: "Unsuccessful operation: indicates that an unexpected, internal server error happened which prevented the node from fulfilling the request."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InternalErrorResponse'
+  '/api/core/v3/commitments/by-index/{index}/utxo-changes':
+    get:
+      tags:
+        - commitments
+      summary: Get all UTXO changes of a given slot by commitment index.
+      description: Get all UTXO changes of a given slot by commitment index.
+      parameters:
+        - in: path
+          name: index
+          schema:
+            type: integer
+          example: 154862
+          required: true
+          description: Index of the commitment to look up.
+      responses:
+        '200':
+          description: "Successful operation."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UTXOChangesResponse'
+              examples:
+                default:
+                  $ref: >-
+                    #/components/examples/get-utxo-changes-response-example
+        '400':
+          description: "Unsuccessful operation: indicates that the provided data is invalid."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BadRequestResponse'
+        '403':
+          description: "Unsuccessful operation: indicates that the endpoint is not available for public use."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ForbiddenResponse'
+        '404':
+          description: "Unsuccessful operation: indicates that the requested data was not found."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NotFoundResponse'
+        '500':
+          description: "Unsuccessful operation: indicates that an unexpected, internal server error happened which prevented the node from fulfilling the request."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InternalErrorResponse'
+
+components:
+  examples:
+    get-routes-response-example:
+      value:
+        routes:
+          - "core/v3"
+          - "dashboard-metrics/v2"
+          - "debug/v2"
+    get-info-response-example:
+      value:
+        name: Example Node
+        version: 1.0.0
+        status:
+          isHealthy: true
+          acceptedTangleTime: '1692966168000000000'
+          relativeAcceptedTangleTime: '1692966173856059205'
+          confirmedTangleTime: '1692966168000000000'
+          relativeConfirmedTangleTime: '1692966173855931045'
+          latestCommitmentID: "0x8fb6cbb1ecf02c6b5a6a5a6a5f6c5a6a5f6c5a6a5a6a5a6a5f6c5a6a5a6a5f6c50000000000000000"
```

```suggestion
          latestCommitmentId: "0x8fb6cbb1ecf02c6b5a6a5a6a5f6c5a6a5f6c5a6a5a6a5a6a5f6c5a6a5a6a5f6c50000000000000000"
```
Looks like it needs to be updated in iota.go as well

---

iota.go is correct now. :+1: